### ISSUE TITLE
Player Leveling

### ISSUE THREAD

**Issue by cmkrist** [OpenChamp/_original_client#46]
Lvl 0 => 250XP Required

XP_Required= Old_XP_Required + 100

- [x] Minions Give XP
- [x] Players Give XP
- [ ] Towers Give XP

**Comment by noah1510** [comment]
This was my suggestion on discord regarding the level system in general:

One of the things making lol very frustrating when behind is the level system.
Each level up is a huge power spike and being 1 or 2 levels down can make many situations unplayable even if the actual exp difference is not that big.

My suggestion would be to increase the amount of levels and decrease the per level up bonus.
I think having 3 or even 5 times the levels lol has would be great.
In that case you would get an ability point every 3/5 levels.
This would make the progression feel more linear instead of the huge bumps each level up in lol gives.
Maybe the ability point should be given instead of level up stats.
This could be an interesting mechanic, since certain champions could delay ability points in favor of raw stats.

When it comes to drops I think kills rewards should be intentionally low.
This would encourage kill to be a means towards gaining a strategic advantage instead of the kill chasing that happens all the time.
Exp and gold from kills should be at a similar level than siege minions maybe boosted if the team is behind.

Exact numbers regarding all of this will require a lot of fine tuning in the future, for now these are just some general idea up for discussion.

**Comment by cmkrist** [comment]
I don't agree at all with this. I think the level spikes are what make LoL fun, especially as you get better at timing  your spikes against the enemy team.

I've made a community poll to see how the community feels about this

**Comment by InAJacket** [comment]
Level timers are a huge part of LOL on a competitive level, even in lower ranks. If we remove the skill behind that, we're inherently punishing people who would otherwise play around it. Personally I'm not against a different number of levels, but the power spikes are a massive part of what makes League so dynamic, especially in the early-game.

**Comment by noah1510** [comment]
You still need to time level ups, if anything this could add even more skill expression.

If every champion has a aphelios style level up this opens very many possibilities. While each level up is less important, the sum of many of these is just as important.

Since you don't gain an ability point every level there is still the need to time certain level ups.

Giving the option to defer the use of skill upgrades in favor of raw stats could also provide interesting options.

**Comment by cmkrist** [comment]
![image](https://github.com/OpenChamp/client/assets/137073012/4816963f-0fbc-45df-9605-15e097603803)
For now lets build the LoL Leveling systemand we can come up with alternative gamemodes with alternative leveling systems at a later time

**Comment by noah1510** [comment]
Levels are implemented this includes stat increases and even exp and gold share for assists (kind of).
Towers are not in the game atm but once implemented should work as well.
